flexmeasures/api/common/utils/api_utils.py

```diff
@@ -1,126 +1,26 @@
 from __future__ import annotations
 
 from timely_beliefs.beliefs.classes import BeliefsDataFrame
-from typing import List, Sequence, Tuple, Union
-import copy
-from datetime import datetime, timedelta
-from json import loads as parse_json, JSONDecodeError
+from typing import List, Sequence, Union
+from datetime import timedelta
 
 from flask import current_app
-from inflection import pluralize
 from numpy import array
 from psycopg2.errors import UniqueViolation
 from rq.job import Job
 from sqlalchemy.exc import IntegrityError
-import timely_beliefs as tb
 
 from flexmeasures.data import db
-from flexmeasures.data.models.assets import Asset, Power
-from flexmeasures.data.models.generic_assets import GenericAsset, GenericAssetType
-from flexmeasures.data.models.markets import Price
-from flexmeasures.data.models.time_series import Sensor, TimedBelief
-from flexmeasures.data.models.weather import WeatherSensor, Weather
-from flexmeasures.data.services.time_series import drop_unchanged_beliefs
-from flexmeasures.data.utils import save_to_session, save_to_db as modern_save_to_db
```

Comment:
It might be time to retire the "modern" labeling here. You're deleting the legacy `save_to_db` from this module in this PR.